skin.json

```diff
@@ -129,7 +129,16 @@
   "closedCaptionOptions": {
     "defaultEnabled": true,
     "defaultLanguage": "en",
-    "availableLanguages": []
+    "availableLanguages": [],
+    "defaultTextColor": "White",
```

Comment:
I just followed the implementation that was already there, I could see how it might help the customer to have "default" before each option.